src/amuse/community/fi/interface.py

```diff
@@ -1,2414 +1,2599 @@
+        function.addParameter("id", dtype="i", direction=function.IN)
+        for x in ["mass", "x", "y", "z", "vx", "vy", "vz", "tform", "radius"]:
+            function.addParameter(x, dtype="d", direction=function.OUT)
+        function.result_type = "i"
         return function
 
-    @legacy_function    
+    @legacy_function
     def set_state():
-        function = LegacyFunctionSpecification()   
+        function = LegacyFunctionSpecification()
         function.can_handle_array = True
-        function.addParameter('id', dtype='i', direction=function.IN)
-        for x in ['mass','x','y','z','vx','vy','vz']:
-            function.addParameter(x, dtype='d', direction=function.IN)
-        function.addParameter('radius', dtype='d', direction=function.IN, default = 0)
-            
-        function.result_type = 'i'
+        function.addParameter("id", dtype="i", direction=function.IN)
+        for x in ["mass", "x", "y", "z", "vx", "vy", "vz"]:
+            function.addParameter(x, dtype="d", direction=function.IN)
+        function.addParameter("radius", dtype="d", direction=function.IN, default=0)
+
+        function.result_type = "i"
         return function
 
-    @legacy_function    
+    @legacy_function
     def set_state_star():
-        function = LegacyFunctionSpecification()   
+        function = LegacyFunctionSpecification()
         function.can_handle_array = True
-        function.addParameter('id', dtype='i', direction=function.IN)
-        for x in ['mass','x','y','z','vx','vy','vz']:
-            function.addParameter(x, dtype='d', direction=function.IN)
-        
-        function.addParameter('tform', dtype='d', direction=function.IN, default = 0)
-        function.addParameter('radius', dtype='d', direction=function.IN, default = 0)
-        function.result_type = 'i'
-        return function
-    
-    @legacy_function    
+        function.addParameter("id", dtype="i", direction=function.IN)
+        for x in ["mass", "x", "y", "z", "vx", "vy", "vz"]:
+            function.addParameter(x, dtype="d", direction=function.IN)
+
+        function.addParameter("tform", dtype="d", direction=function.IN, default=0)
+        function.addParameter("radius", dtype="d", direction=function.IN, default=0)
+        function.result_type = "i"
+        return function
+
+    @legacy_function
     def set_state_sph():
-        function = LegacyFunctionSpecification()   
+        function = LegacyFunctionSpecification()
         function.can_handle_array = True
-        function.addParameter('id', dtype='i', direction=function.IN)
-        for x in ['mass','x','y','z','vx','vy','vz','u',]:
-            function.addParameter(x, dtype='d', direction=function.IN)
-        function.addParameter('h_smooth', dtype='d', direction=function.IN, default = 0)
-        function.result_type = 'i'
-        return function
-    
-    @legacy_function    
+        function.addParameter("id", dtype="i", direction=function.IN)
+        for x in [
```

Comment:
Ouch! Oh well, it is what it is with autoformatters...

Reply:
It makes sense - the autoformatter would add spaces, and with the spaces I guess it exceeds the max line length (which defaults to 80 rather than 88 - may have to change that).

Reply:
88 is surprisingly much better than 80, so let's change it indeed. And maybe do it right away and reformat this and then merge it, so that we don't have to do all this again later.